### ISSUE TITLE
[Improvement] Unable to install

### ISSUE THREAD

**Issue by LukeSavefrogs** [ibm-messaging/mq-dev-samples#13]
The following is a problem i faced during the installation via the [script](https://github.com/ibm-messaging/mq-dev-samples/blob/master/gettingStarted/installing-mq-ubuntu/mq-ubuntu-install.sh). I think it's worth metioning in the `README.md` to help newcomers.
 
## Problem
Installation via script fails when the path is inside the `/root/` folder.
```
root@dev-server:/etc/apt/sources.list.d# sudo apt update
Get:1 file:/root/MQServer ./ InRelease
Ign:1 file:/root/MQServer ./ InRelease

[...]

Reading package lists... Done
N: Download is performed unsandboxed as root as file '/root/MQServer/./InRelease' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)
E: Failed to fetch file:/root/MQServer/./Packages  File not found - /root/MQServer/./Packages (2: No such file or directory)
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

## Solution
Launch the script in a folder accessible by the user `_apt` (such as `/tmp`).
```
root@dev-server:/tmp# bash < <(curl -s https://raw.githubusercontent.com/ibm-messaging/mq-dev-samples/master/gettingStarted/installing-mq-ubuntu/mq-ubuntu-install.sh)
```

**Comment by chughts** [comment]
Resolved in #15

**Comment by LukeSavefrogs** [comment]
Thank you @chughts 🎉
